spec/features/staff_can_create_a_comment_spec.rb

```diff
@@ -65,7 +65,7 @@
     end
   end
 
-  context 'when the defect is for a communal communal_area' do
+  context 'when the defect is for a communal area' do
```

Comment:
Does this need to be committed?

Reply:
I believe this was a typo, we use `communal area` not `communal communal_area`. Happy to leave off this PR but thought I'd sneak it in.